lib/common/mock.c

```diff
@@ -70,15 +70,22 @@ __wrap_calloc(size_t nmemb, size_t size)
  * If pcmk__mock_getenv is set to true, later calls to getenv() must be preceded
  * by:
  *
+ *     expect_*(__wrap_getenv, name, ...);
  *     will_return(__wrap_getenv, return_value);
+ *
+ * expect_* functions: https://api.cmocka.org/group__cmocka__param.html
  */
 
 bool pcmk__mock_getenv = false;
 
 char *
 __wrap_getenv(const char *name)
 {
-    return pcmk__mock_getenv? mock_ptr_type(char *) : __real_getenv(name);
+    if (!pcmk__mock_getenv) {
+        __real_getenv(name);
```

Comment:
Should be `return`. Will fix.

Reply:
(making a note that no regression test catches this; does `mock.c` even get pulled into a regular build?)

Reply:
No, mock.c is only built into libcrmcommon_test as part of "make check".